lib/ecl/ecl_sum_data.c

```diff
@@ -526,15 +526,15 @@ bool ecl_sum_data_check_sim_days( const ecl_sum_data_type * data , double sim_da
 
 
 static int ecl_sum_data_get_index_from_sim_time( const ecl_sum_data_type * data , time_t sim_time) {
-  time_t data_start_time = time_interval_get_start( data->sim_time );
-  time_t sim_end         = time_interval_get_end( data->sim_time );
+  time_t sim_start = time_interval_get_start(data->sim_time);
```

Comment:
There is a conceptual difference between `data_start_time` and `sim_start`. The summary case will always have a correct value for `sim_start` - but for restarted simulations there can be a finite time interval: `[sim_start, data_start_time]` where we do not have data. Hence I would prefer to keep the name `data_start_time`.